packages/dune-private-libs/dune-private-libs.2.0.1/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Private libraries of Dune"
+description: """
+!!!!!!!!!!!!!!!!!!!!!!
+!!!!! DO NOT USE !!!!!
+!!!!!!!!!!!!!!!!!!!!!!
+
+This package contains code that is shared between various dune-xxx
+packages. However, it is not meant for public consumption and provides
+no stability guarantee.
+"""
+maintainer: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+license: "MIT"
+homepage: "https://github.com/ocaml/dune"
+doc: "https://dune.readthedocs.io/"
+bug-reports: "https://github.com/ocaml/dune/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.07"}
```

Comment:
```suggestion
  "ocaml" {>= "4.06"}
```
I believe this should be 4.06, otherwise when dune.2.0.0 becomes unavailable, people on 4.06 won't be able to install dune2 anymore if they have packages relying on dune-configurator for instance, and they will revert to dune.1.11.4